web-bundle/src/main/resources/com/graphhopper/maps/js/main-template.js

```diff
@@ -78,17 +78,21 @@ $(document).ready(function (e) {
     // todo: the idea was to highlight everything so if we start typing the example is overwritten. But unfortunately
     // this does not work. And not even sure this is so useful?
     showCustomModelExample();
-    cmEditor.validListener = function(valid) {
+    cmEditor.validListener = function (valid) {
         $("#custom-model-search-button").prop('disabled', !valid);
     };
-    $("#custom-model-button").click(function() {
+    ghRequest.cmEditor = cmEditor;
```

Comment:
ghRequest now keeps a reference to the custom model editor. rather hacky, but it 'works' :)